server/services/cardsManager.js

```diff
@@ -1,4 +1,4 @@
-module.exports.assign = (users, number_of_cards) => {
+exports.assign = (users, number_of_cards) => {
```

Comment:
Not a fan of the underscore naming convention here, can it be changed to camel case?